warehouse/core/src/test/resources/test-root-rules.xml

```diff
@@ -0,0 +1,26 @@
+<ageoffConfiguration>
```

Comment:
@brianloss @ivakegg  This is an example of a root config that would contain all of the core configs and rules, test-customized-rules.xml is the example of the child that overrides some of these